api/pkg/backend/service.go

```diff
@@ -47,6 +48,9 @@ const (
 	dataflowService_K8S    = "soda.multicloud.v1.dataflow"
 )
 
+// List of object storge providers supported by s3 services
+var object_types = []string{"aws-s3", "azure-blob", "ibm-cos", "hw-obs", "ceph-s3", "gcp-s3", "fusionstorage-object", "yig", "alibaba-oss", "sony-oda"}
```

Comment:
var objectStoarge = map[string]struct{}{
                                   "aws-s3": struct{}{},
                                    "azure-blob": struct{}{},
                                    "ibm-cos": struct{}{}, 
                                    "hw-obs": struct{}{},
                                     "ceph-s3": struct{}{},
                               }
func isObjectStorage(storage string) bool {
     _, ok := objectStoarge[storage ]
     return ok
}

IMHO, Above might be better implementation

Reply:
Thanks for the suggestion. Yes optimized it now. o(n) -> o(1) :)